tests/Behat/Page/Admin/OrderRefundsPage.php

```diff
@@ -35,7 +35,16 @@ public function pickUnitWithProductToRefund(string $productName, int $unitNumber
     {
         $units = $this->getUnitsWithProduct($productName);
 
-        $units[$unitNumber]->find('css', '.checkbox input')->check();
+        $value = $units[$unitNumber]->find('css','td:nth-child(2)');
+
+        $refunded = substr($this->getUnitRefundedTotal($value), 1);
+
+        $total = substr($this->getUnitTotal($value), 1);
+
+        /** @var double $total */
```

Comment:
Is it needed?

Reply:
I think that if we get int and double variables, we lost the sum after the decimal point? I think well?